pages/onboarding.html

```
@@ -196,22 +196,24 @@ <h3>Consensus</h3>
             apply is a <strong>privilege, not a right.</strong></span></p>
         </div>
         <div class="visual">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="visual"> ] on line 198.

(tag-pair)